src/arch/host/configs/library_defconfig

```diff
@@ -3,6 +3,7 @@ CONFIG_COMP_CROSSOVER=y
 CONFIG_COMP_DCBLOCK=y
 CONFIG_COMP_DRC=y
 CONFIG_COMP_FIR=y
+CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
```

Comment:
is google RTC processing is a common feature or platform specific feature? 
if it is platform specific, I  would suggest put it to platform config.

Reply:
The name of this config file is perhaps misleading, though there's "host" in the path. This is for testbench and it's safe to have every component included with it. Testbench build is a good catcher in CI for build errors since we have many components with "default n" in kconfig those might be missed in normal platform builds.

Reply:
Don't you want to have the mock config as well here ?